app/models/setting.rb

```diff
@@ -32,11 +32,14 @@ class Setting < Translation
            value:      { type: :dynamic },
            prefix:     { type: :hidden },
            label:      { writable_method: :god? },
-           role:       { type: :select, data: Admin::ROLES }
+           role:       { type: :select, data: Admin::ROLES },
+           is_proc:    { type: :boolean },
+           images:     { type: :inline }
+
     config :admin do
       index fields: [:label],
             title: "Settings"
-      form  fields: [:label, :field_type, :prefix, :key, :value, :hint, :role, :is_proc, :images],
```

Comment:
We aren't worrying about `:images` for this release, but once redesign comes in it will work again, so let's have it ready for that.